packages/rest/test/acceptance/bootstrapping/application.acceptance.ts

```diff
@@ -102,25 +109,28 @@ describe('Bootstrapping the application', () => {
           };
         }
       }
-
       const app = new Application({
         components: [ConfigComponent, GreetingComponent],
       });
+      const server = new RestServer(app);
+      app.server(server);
 
-      expect(app.getSync('greeting')).to.equal('Hi!');
+      expect(server.getSync('greeting')).to.equal('Hi!');
```

Comment:
I think the changes in this test should be reverted - components are bound at application level now, so we should not require any server instance.

❗️ In fact, I think most of this file should stay back in core with no changes. It's called `application.acceptance.ts` for a reason, so to speak.

AFAICT, the only server-specific part is Sequence related tests and "starting the application" below, these should go to `server.acceptance.ts` in `packages/rest`.

Thoughts?